src/python/pants/init/engine_initializer.py

```diff
@@ -113,13 +115,29 @@ def _apply_default_sources_globs(base_class, target_type):
 _apply_default_sources_globs(RemoteSourcesAdaptor, RemoteSources)
 
 
-def _legacy_symbol_table(build_file_aliases: BuildFileAliases) -> SymbolTable:
+def _make_target_adaptor(
```

Comment:
This was moved up from lower in the file. There were no changes other than type hints + type ignore comments.